pydoc/getting_started/tutorial/example.md

```diff
@@ -0,0 +1,151 @@
+# Example
```

Comment:
How to DLitefying existing classes should not be an example, but a separate point in the user guide.